accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -112,10 +123,310 @@ Windows-specific:
 
 * **DpapiProtectedConfigurationProvider**
 
+### System.Data.SqlTypes
+
+* **SqlFileStream**
+    - `Flush()`
+    - `get_Name()`
+    - `Read(Byte[], Int32, Int32)`
+    - `Write(Byte[], Int32, Int32)`
+    - `.ctor(String, Byte[], FileAccess)`
+    - `.ctor(String, Byte[], FileAccess, FileOptions, Int64)`
+    - `get_CanRead()`
+    - `get_CanSeek()`
+    - `get_CanWrite()`
+    - `get_Length()`
+    - `get_Position()`
+    - `get_TransactionContext()`
+    - `Seek(Int64, SeekOrigin)`
+    - `set_Position(Int64)`
+    - `SetLength(Int64)`
+
+### System.Data.OleDb
```

Comment:
Should we mark the entire assembly?

Reply:
@ajcvickers does this make sense to you?

Reply:
Yes, mark the entire assembly.